docs/versioned_docs/version-7.0.x/configuration/overview.md

```diff
@@ -423,6 +425,93 @@ http:
         query: "/oauth2/sign_in"
 ```
 
+### ForwardAuth with static upstreams configuration
+
+Redirect to sign_in functionality provided without the use of `errors` middleware with [Traefik v2 `ForwardAuth` middleware](https://doc.traefik.io/traefik/middlewares/forwardauth/) pointing to oauth2-proxy service's `/` endpoint
+
+**Following options need to be set on `oauth2-proxy`:**
+- `--upstream=static://202`: Configures a static response for authenticated sessions
+- `--reverseproxy=true`: Enables the use of `X-Forwarded-*` headers to determine redirects correctly
+
+```yaml
+http:
+  routers:
+    a-service-route-1:
+      rule: "Host(`a-service.example.com`, `b-service.example.com`) && PathPrefix(`/`)"
+      service: a-service-backend
+      middlewares:
+        - oauth-auth-redirect # redirects all unauthenticated to oauth2 signin
+      tls:
+        certResolver: default
+        domains:
+          - main: "example.com"
+            sans:
+              - "*.example.com"
+    a-service-route-2:
+      rule: "Host(`a-service.example.com`) && PathPrefix(`/no-auto-redirect`)"
+      service: a-service-backend
+      middlewares:
+        - oauth-auth-wo-redirect # unauthenticated session will return a 401
+      tls:
+        certResolver: default
+        domains:
+          - main: "example.com"
+            sans:
+              - "*.example.com"
+    oauth:
+      rule: "Host(`a-service.example.com`, `b-service.example.com`, `oauth.example.com`) && PathPrefix(`/oauth2/`)"
```

Comment:
Where's `oauth.example.com/` handled?

Reply:
🤦 , looks like I missed it during the refactor, added it now, thanks